### PR TITLE
Render deploy script and additional tweaks

### PR DESCRIPTION
**Pull request by majordwarf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,62 @@
...
 
-## About The Project
 Terasology’s basic engine can be extended by a huge amount (201 right now!) of modules. Keeping track of them is not the easiest task. This is why this gatsby framework provides an automated generator for a website, listing all of them. This showcase website will increase discoverability by allowing to filter and search for modules by keywords and categories.
 
-### Built With
+<h2 id="built-with">Built With</h2>
+
 The following generator is built using the following software, you'll need them installed in your workspace to run properly. 
 * [Node](https://nodejs.org/en/)
 * [Gatsby](https://www.gatsbyjs.org/)
```
Comment (Cervator) [inline README.md:28]: Minor: Both Node and Yarn have extra (if minimal) steps about installing them, yet Gatsby doesn't. Consistency good :-)

For me on my Mac it was just `npm install -g gatsby-cli` after I had installed Node and Yarn

File: .gitignore [hunk 2]
```diff
@@ -59,4 +59,5 @@ public/
 bundle-stats.json
 
 # Generated JSON
-src/generated/result.json
\ No newline at end of file
+src/generated/blog-result.json
+src/generated/module-result.json
```
Comment (Cervator) [inline .gitignore]: Trivial: best practice to have a newline at the end of text files

File: modules/Gooey Example/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 posttype: "module"
 title: "Gooey Example"
```
Comment (Cervator) [inline modules/Gooey Example/index.md:3]: Should spaces be allowed in module names when used as a directory? Could that cause trouble if it ends up in a URL?

**Review by Cervator — APPROVED** [review]
Checked it out locally and noticed a couple things. There was an error about a blog result on startup, is that expected?

```
success building schema — 0.228 s
error gatsby-node.js returned an error


  Error: ENOENT: no such file or directory, open './src/generated/blog-result.json'
  
  - gatsby-node.js:133 Object.exports.createPages
    /Users/cervator/dev/terasology/git_ws/terasology.github.io/gatsby-node.js:133:6
  

success createPages — 0.047 s
```

Also in the footer of the site I noticed some odd indentation issues, unless "Forum" and "License" are meant to be indented more? Also I'm not what the "Terasology, Moving Blocks!" is really doing, was that from an old site? Meant as a tag line or? :-) "The Terasology Foundation" instead maybe? "MovingBlocks" is mostly out of date

Screen Shot 2019-09-01 at 5 08 55 PM

Maybe we could also have some notes in the readme about how to stage some module data? I got nuthin' :-)

Same story for the Render bits. I see related files but just reading the readme I have no idea what to do to test it out.

In any case from what I can see and test - works!

File: modules/Gooey Example/index.md (hunk 3 above)
Comment (majordwarf) [reply modules/Gooey Example/index.md:3]: It won't matter as the the url are generated via slugs that will put a hypen in space of space.

**Pull request merged**